library.properties

```
@@ -2,8 +2,8 @@ name=Arduino_MachineControl
 version=1.1.1
 author=Arduino
 maintainer=Arduino <info@arduino.cc>
-sentence=Arduino Library for Portenta Machine Control - PMC
-paragraph=
+sentence=DEPRECATED. Arduino Library for Portenta Machine Control - PMC
+paragraph=This library is no longer actively maintained and is deprecated. Consider migrating to <a href="https://github.com/arduino-libraries/Arduino_PortentaMachineControl">Arduino_PortentaMachineControl</a> library.
```

```suggestion
paragraph=This library is no longer actively maintained and is deprecated. Consider migrating to the Arduino_PortentaMachineControl library.
```

HTML is not supported by the Arduino IDE 2.x Library Manager: https://github.com/arduino/arduino-ide/issues/644. The library name will be sufficient because most users will see this in Library Manager and so can simply type `Arduino_PortentaMachineControl` into the search field.